sklearn/metrics/tests/test_pairwise_distances_reduction.py

```diff
@@ -564,6 +564,10 @@ def test_pairwise_distances_reduction_is_usable_for():
     assert not BaseDistancesReductionDispatcher.is_usable_for(
         X_csr, Y_csr, metric="sqeuclidean"
     )
+    # TODO: add this assertion once #24542 and #24715 are merged. 
```

Comment:
Ah, the previous suggestion introduced an extra space, making `black` stream on the CI.
```suggestion
    # TODO: add this assertion once #24542 and #24715 are merged.
```